x-pack/plugins/siem/public/components/empty_page/index.tsx

```diff
@@ -4,34 +4,73 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import { EuiButton, EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem, IconType } from '@elastic/eui';
 import React from 'react';
 import { pure } from 'recompose';
 import styled from 'styled-components';
 
+const EmptyPrompt = styled(EuiEmptyPrompt)`
+  align-self: center; // Corrects horizontal centering in IE11
+`;
+
 interface EmptyPageProps {
-  message: string;
-  title: string;
-  actionLabel: string;
-  actionUrl: string;
+  actionPrimaryIcon?: IconType;
```

Comment:
nit - I was wondering if this should be in an array object with a specific interface. therefore we do not have to define so many props.

Reply:
If it was something I was mapping, then I could see the value in switching to an array of objects. But since I just have two distinct buttons (one button being optional), I wonder if it's simpler the way I have it. I'm happy to change it if what I have is bad practice, but what do you think?

Reply:
it was just a `nit`, but as right now, we can keep it like that but if we add to add more I would move to array